cli/internal/globby/globby.go

```diff
@@ -15,49 +15,54 @@ import (
 var _aferoOsFs = afero.NewOsFs()
 var _aferoIOFS = afero.NewIOFS(_aferoOsFs)
 
-func GlobFiles(basePath string, includePatterns []string, excludePatterns []string) []string {
+// GlobFiles returns an array of files that match the specified set of glob patterns.
+func GlobFiles(basePath string, includePatterns []string, excludePatterns []string) ([]string, error) {
 	return globFilesFs(_aferoIOFS, basePath, includePatterns, excludePatterns)
 }
 
-// isRelativePath ensures that the the requested file path is a child of `from`.
-func isRelativePath(from string, to string) (isRelative bool, err error) {
+// checkRelativePath ensures that the the requested file path is a child of `from`.
+func checkRelativePath(from string, to string) error {
```

Comment:
There was no reason for this to have two return values, only one of them was ever needed to be checked. I've combined them, changed the name, and dropped the return argument names.